config.rb

```diff
@@ -1,5 +1,6 @@
 require 'govuk_tech_docs'
 
+# require "govuk_tech_docs/path_helpers"
```

Comment:
This does not seem to be working, fails with `'require': cannot load such file -- govuk_tech_docs/path_helpers (LoadError)`